frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link"
+
+const Footer = () => {
+  return (
+    <footer className="bg-secondary bg-opacity-30 text-light">
+      <div className="max-w-7xl mx-auto py-12 px-4 sm:px-6 lg:px-8">
+        <div className="flex justify-center space-x-6">
+          <nav className="flex gap-10 font-medium text-sm">
+            <Link href="/get-started">Get started</Link>
+            <Link href="/how-it-works">How does it works?</Link>
+            <Link href="/about-us">About us</Link>
+            <Link href="/terms-of-service">Terms of Service</Link>
+            <Link href="/privacy-policy">Privacy Policy</Link>
+          </nav>
+        </div>
+        <div className="mt-8 text-center text-sm">
+          <p>&copy; 2021 Festwrap. All rights reserved.</p>
```

Comment:
Probably 2024, right? 😅

Reply:
Yes, I've generated some of the code with v0 to try the platform and I didn't see this mistake. I change it now.